packages/protocol/contracts/governance/Election.sol

```diff
@@ -339,12 +339,60 @@ contract Election is
     return true;
   }
 
+  /**
+   * @notice Decrements `value` pending or active votes for `group` from `account`.
+   *         First revokes all pending votes and then, if `value` votes haven't
+   *         been revoked yet, revokes additional active votes.
+   *         Fundamentally calls `revokePending` and `revokeActive` but only resorts groups once.
+   * @param account The account whose votes to `group` should be decremented.
+   * @param group The validator group to decrement votes from.
+   * @param value The number of votes to decrement and revoke.
+   * @param lesser The group receiving fewer votes than the group for which the vote was revoked,
+   *               or 0 if that group has the fewest votes of any validator group.
+   * @param greater The group receiving more votes than the group for which the vote was revoked,
+   *                or 0 if that group has the most votes of any validator group.
+   * @param index The index of the group in the account's voting list.
+   * @return uint256 Number of votes successfully decremented and revoked, with a max of `value`.
+   */
+  function _decrementVotes(
+    address account,
+    address group,
+    uint256 value,
+    address lesser,
+    address greater,
+    uint256 index
+  ) internal returns (uint256) {
+    require(group != address(0) && 0 < value);
```

Comment:
being an internal function, it seems that the check are not needed..
group is surely != than 0.
And value > 0 should probably be checked on the outer loop